src/bip85.rs

```diff
@@ -0,0 +1,49 @@
+use bdk::bitcoin::{
+    hashes::{sha512, Hash, HashEngine, Hmac, HmacEngine},
+    secp256k1::Secp256k1,
+    util::bip32::{ChildNumber, DerivationPath, ExtendedPrivKey},
+};
+
+pub fn get_bip85_bytes<const L: usize>(
+    xpriv: ExtendedPrivKey,
+    app_num: u32,
```

Comment:
I shouldn't be allowed to pass in the `app_num` here when it's fixed to `128169` for `L` bytes of hex. I know that technically we are not returning `hex` from the function but that's just an implementation detail. It looks like you are trying to implement do a more general `bip85` but we only want the `hex` bit of it.